crates/red_knot_python_semantic/src/semantic_index/definition.rs

```diff
@@ -466,6 +467,31 @@ pub enum DefinitionKind<'db> {
 }
 
 impl DefinitionKind<'_> {
+    pub(crate) fn range(&self) -> TextRange {
```

Comment:
would it be better to implement `Ranged` for `DefinitionKind`? Then we'd also get the `start()` and `stop()` methods that the trait implements

Reply:
I thought about that but that would make it publicly available which I wasn't sure about. I could possibly rename this function to `debug_range` and provide a brief docs on what this represents.

I guess it's fine to implement it, will do so.